src/EditorFeatures/Core/Implementation/Diagnostics/AbstractDiagnosticsTaggerProvider.AggregatingTagger.cs

```diff
@@ -275,19 +512,12 @@ private void OnDiagnosticsUpdatedOnForeground(DiagnosticsUpdatedArgs e)
                 // Note: if the Solution or Document is null here, then that means the document
                 // was removed.  In that case, we do want to proceed so that we'll produce 0
                 // tags and we'll update the editor appropriately.
-                SourceText sourceText = null;
-                ITextSnapshot editorSnapshot = null;
-                if (e.Solution != null)
-                {
-                    var diagnosticDocument = e.Solution.GetDocument(e.DocumentId);
-                    if (diagnosticDocument != null)
-                    {
-                        if (!diagnosticDocument.TryGetText(out sourceText))
+                        if (!diagnosticDocument.TryGetText(out var sourceText))
```

Comment:
❔ Indentation change here?

Reply:
This was kept to make the PR easier to review.  once done i can dedent :)